trace/Sampling.md

```diff
@@ -8,7 +8,31 @@ exporters.
 
 The Sampling bit is always set only at the start of a Span, using a `Sampler`
 
-### What kind of samplers does OpenCensus support?
+### Sampler interface
+A `Sampler` is an interface with only one method `sample` that accepts a set of sampling
```

Comment:
> only one method `sample`

Let's not give this much of low-level implementation details. It is enough to say Sampler should be an interface that gives a sampling decision based on given sampling parameters.

Reply:
Done.